zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -109,7 +110,10 @@ class ZiplineLoader internal constructor(
       maxSizeInBytes = maxSizeInBytes,
       nowEpochMs = nowEpochMs,
     )
-    cache.initialize()
+    // TODO: make ZiplineLoader.withCache() defer initialization or launch it
+    runBlocking {
+      cache.initialize()
+    }
```

Comment:
Yeah this will prevent the strict mode crash, but it does so by tricking strict mode by blocking on the same work moved but moved to a background thread.

I would launch this in an `async` and then save the `Deferred` such that whenever you need to access the cache you call `await()` which will either return synchronously if the initialization finished or it will suspend waiting for it to finish.